samples/caf_sensor_manager/prj.conf

```diff
@@ -18,6 +18,10 @@ CONFIG_APP_EVENT_MANAGER=y
 CONFIG_REBOOT=y
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 
+# Using simulated sensor (the DK does not have built-in sensor)
+CONFIG_SENSOR_SIM=y
+CONFIG_SENSOR_STUB=n
```

Comment:
You are changing CONFIG_SENSOR_SIM in two commits it can be confusing.

Reply:
I think it is fixed now.